guides/workload-variant-autoscaling/workload-variant-autoscaler/values.yaml

```diff
@@ -0,0 +1,51 @@
+wva:
```

Comment:
Make sure the package is renamed from workload-varient-autoscaler -> workload-autoscaling

Reply:
done in latest push